static/docs/commands-reference/get.md

```diff
@@ -0,0 +1,45 @@
+# get
+
+Download or copy file or directory from another DVC repository into the local
+file system.
+
+> Unlike `dvc import`, this command does not track the downloaded data file(s)
+> (does not create a DVC-file).
+
+## Synopsis
+
+```usage
+usage: dvc get [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
+
+positional arguments:
+  url                   DVC repository URL to download data from.
```

Comment:
we should update this, it's not clear that DVC repo is Git repo. And it's easier to get the concept when you see that you can pass a link to a Git server.

Reply:
Ok but that's a change for the core `dvc` repo. So I'll update here and open an issue there...

Reply:
Ok updated in f3f57c1. What do you think about the text? `DVC repository URL (Git server link).`

Reply:
how about - `Git repository URL with DVC project to download from`?

Reply:
Yes, better. Went for `URL of Git repository with DVC project to download from` in c4444c6.